### PR TITLE
Initial Commit - initial controller and pipeline

### PR DESCRIPTION
**Pull request by ianco**
Initial issuer controller and agent
Template for data pipeline

### CODE REVIEW

**Comment by WadeBarnes** [comment]
@ianco, There are still some references to EAO in the code, pipelines, and documentation.

Other than that, I've tested this against the corresponding changes to [indy-catalyst](https://github.com/bcgov/indy-catalyst/pull/292) using the credential registry starter kit.

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+      - AGENT_IN_PORT_1=${AGENT_IN_PORT_1}
+      - AGENT_IN_PORT_2=${AGENT_IN_PORT_2}
+      - AGENT_IN_PORT_3=${AGENT_IN_PORT_3}
+      - AGENT_ADMIN_PORT=${AGENT_ADMIN_PORT}
+      - AGENT_NAME=${AGENT_NAME}
+    networks:
+      - orgbook
+      - myorg
+    ports:
+      - ${AGENT_IN_PORT_1}:${AGENT_IN_PORT_1}
```
Comment (nrempel) [inline docker/docker-compose.yml]: Minor nit, but these could be renamed to more descriptive names like "INBOUND_TRANSPORT_HTTP_PORT" etc

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+      - ${AGENT_ADMIN_PORT}:${AGENT_ADMIN_PORT}
+    depends_on:
+      - myorg-wallet-db
+    entrypoint: /bin/bash
+    command: [
+        "-c",
+        " sleep 5; \
+        aca-py \
+        --inbound-transport http '0.0.0.0' ${AGENT_IN_PORT_1} \
+        --inbound-transport http '0.0.0.0' ${AGENT_IN_PORT_2} \
```
Comment (nrempel) [inline docker/docker-compose.yml]: There is an example of running two separate inbound transports of the same type, but isn't really necessary in this situation. This line can be removed.

**Review by nrempel — CHANGES_REQUESTED** [review]
Added just a few minor comments.

The rest looks good. I didn't go too deep on the logic in some modules but I didn't see any issues.

**Review by nrempel — APPROVED** [review]

**Comment by nrempel** [comment]
👌🏼

**Pull request merged**